src/layers/FeatureGroup.js

```diff
@@ -19,9 +19,9 @@ export const FeatureGroup = L.FeatureGroup.extend({
         const { buffer, bufferStyle, data, pane, opacity, isVisible } = this.options;
 
         if (buffer) {
-            const bufferLayer = new Circles({
+            const bufferLayer = new Buffer({
                 pane: `${pane}-buffer`,
-                radius: buffer,
+                buffer,
```

Comment:
Ah I see.  I guess this is the only place this is used?

Reply:
Yes it is.